### ISSUE TITLE
calendar app lacks contrast

### ISSUE THREAD

**Issue by pirey** [EliverLara/Nordic#118]
In calenar app, the foreground color is too similar with background color for each event, which make it kinda hard to tell.
I'm using ubuntu 20.04
![Screenshot from 2021-02-25 17-05-51](https://user-images.githubusercontent.com/10344542/109137572-ed47d780-778b-11eb-827e-db1db5073201.png)

**Comment by EliverLara** [comment]
Hi, That's an app issue as you can see [here](https://gitlab.gnome.org/GNOME/gnome-calendar/-/issues/580), anyway, I've updated the theme with a patch for that.

**Comment by sarsorus** [comment]
The appearance was still unpractical in Monthly and Yearly view for this app.
I'm very unfamiliar with pull requests, I hope you don't mind me suggesting a fix here. This could be fixed by editing `gtk-3.0/gtk.css` around line 5730's the original
```
/*******************
 * Calendar events *
********************/
.color-light label, .color-light.timed label {
  color: #2e3440;
  opacity: 1; }
```
into
```
/*******************
 * Calendar events *
********************/
.color-light label{ opacity: 1}
.color-light.vertical label { color: #8e95a4; }
.color-light.horizontal label, .color-light.vertical.timed label { color: #2e3440; }
.color-light.horizontal.timed label { color: #8e95a4; }
```
(I tried using `:not()` to spare a line but I couldn't make it work)
